### PR TITLE
RESOURCE-557 Soumyo/adding the missing attributes in sqs queue

### PR DESCRIPTION
**Pull request by soumyo13**
### Description

1. Adding the encrypted attribute in aws_sqs_queue
2. Implemented the policy.
3. Implemented policy checks.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [ ] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [ ] All Unit Tests pass
- [ ] `rake lint` passes
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by sathish-progress — APPROVED** [review]

**Review by sa-progress — APPROVED** [review]

File: docs-chef-io/content/inspec/resources/aws_sqs_queue.md [hunk 1]
```diff
@@ -69,6 +68,12 @@ end
 `kms_data_key_reuse_period_seconds`
 : Returns the length of time, in seconds, for which Amazon SQS can reuse a data key to encrypt or decrypt messages before calling AWS KMS again.
 
+`sqs_managed_enabled`
+: Returns information about whether the queue is using SSE-SQS encryption using SQS owned encryption keys.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_sqs_queue.md:72]:
```suggestion
: Returns information about whether the queue is using SSE-SQS encryption using SQS-owned encryption keys.
```

File: docs-chef-io/content/inspec/resources/aws_sqs_queue.md [hunk 2]
```diff
@@ -122,20 +127,62 @@ end
...
 describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueueWhichDoesntExist') do
   it { should_not exist }
 end
 ```
 
+### policy_statement_principal_all_permissions_enabled
+
+The control will pass if the describe returns at least one result.
+
+Use `should` to test the entity should exist.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_sqs_queue.md:150]: This should explain what the result is. Maybe:

```suggestion
The control will pass if at least one Principal has all permissions enabled.

Use `should` to test when a Principal with all permissions should exist.
```

Or something like that. I'm not sure of the context here.

File: docs-chef-io/content/inspec/resources/aws_sqs_queue.md [hunk 3]
```diff
@@ -122,20 +127,62 @@ end
...
+
+```ruby
+describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
+  it { should be_policy_statement_principal_all_permissions_enabled }
+end
+```
+
+Use `should_not` to test the entity should not exist.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_sqs_queue.md:158]:
```suggestion
Use `should_not` to test that a Principal with all permissions should not exist.
```

File: docs-chef-io/content/inspec/resources/aws_sqs_queue.md [hunk 4]
```diff
@@ -122,20 +127,62 @@ end
...
+Use `should` to test the entity should exist.
+
+```ruby
+describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
+  it { should be_policy_statement_action_all_permissions_enabled }
+end
+```
+
+Use `should_not` to test the entity should not exist.
+
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_sqs_queue.md:179]: Same with this

```suggestion
The control will pass if at least one action has all permissions enabled.

Use `should` to test that at least one action has all permissions enabled should exist.

```ruby
describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
  it { should be_policy_statement_action_all_permissions_enabled }
end
```

Use `should_not`  to test that an action with all permissions enabled should not exist.

```

**Review by IanMadd — COMMENTED** [review]
The descriptions here could be a bit more clear

**Review by IanMadd — APPROVED** [review]

**Pull request merged**